File: C_Programming/Day_4/program18.c
//Description : Write a program that accepts number from the user & display its digits

#include<stdio.h>
#include<stdbool.h>

void DisplayDigits(int iNo)
{
    int iDigit = 0;

    while(iNo != 0)
    {
        iDigit = iNo % 10;
        printf("%d\t",iDigit);
        iNo = iNo / 10;
    }

}

int main()
{
    int iValue = 0;

    printf("Enter the number : ");
    scanf("%d",&iValue);

    DisplayDigits(iValue);

    return 0;
}

File: Python_Programming/Day_4/program18.py
# Description : Write a program tha accepts number from the user & display its digits

def DisplayDigits(No):
    Digit = 0

    while(No != 0):
        Digit = No % 10
        print(Digit , "\t" , end = "")
        No = No // 10

def main():

    Value = 0

    print("Enter the number : ")
    Value = int(input())

    DisplayDigits(Value)

if __name__ == "__main__":
    main()